src/main/java/uk/gov/hmcts/reform/sendletter/entity/Letter.java

```diff
@@ -116,6 +116,10 @@ public boolean isFailed() {
         return isFailed;
     }
 
+    public void hasFailed(boolean isFailed) {
```

Comment:
`is`?

Reply:
either `hasFailed` or `setIsFailed`, whichever more appropriate for boolean setter